tests/modules/js-and-css/embed-optimizer/embed-optimizer-test.php

```diff
@@ -131,4 +130,32 @@ public function get_data_to_test_filter_oembed_html_data(): array {
 			),
 		);
 	}
+
+	/**
+	 * Test printing the script lazy-loader.
+	 *
+	 * @covers ::embed_optimizer_lazy_load_scripts
+	 * @dataProvider get_data_to_test_filter_oembed_html_data
```

Comment:
this dataProvider isn't used here, right?
```suggestion
```

Reply:
Good catch